### PR TITLE
Fixed some translation to Portuguese (arvore Binaria) - Binary Search

### PR DESCRIPTION
**Pull request by myluco**
I read through the translation for arvore binaria in the Portuguese section of the guides.

- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [ x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/portuguese/algorithms/binary-search-trees/index.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ localeTitle: Árvores de busca binária
 
 Uma árvore é uma estrutura de dados composta de nós que possui as seguintes características:
 
-1.  Cada árvore tem um nó raiz (no topo) tendo algum valor.
+1.  Cada árvore tem um nó raiz (no topo) quem possui algum valor.
```
Comment (thefrontendwizard) [inline guide/portuguese/algorithms/binary-search-trees/index.md]: Acho que o correto aqui é `que` ao inves de `quem`.

**Review by frontendwizard — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thefrontendwizard This PR has gone stale. Can you either close this PR or make the changes on behalf of the user?

Thank you

**Comment by RandellDawson** [comment]
@myluco Why did you close this PR?  Why not just make the suggested changes?

**Comment by myluco** [comment]
Sorry. I am not familiar how these pull request work... Not even sure what is going on....
I thought you were asking me to close. so I did.
Looking at the request, the portuguese changes do not seem correct: I would not change the "tendo" with "quem possui"
Let me know what you want me to do (unfortunately, I do not have much time this week :( )

**Comment by RandellDawson** [comment]
@myluco Is it that the Portuguese translation you are suggesting vs @thefrontendwizard's suggestion a matter of European vs. Brazilian Portuguese?

**Comment by myluco** [comment]
I know brazilian portuguese. The suggestion from the frontendwizard does not look good from brazilian portuguese perspective. It seems that I did not suggest changes for that part that frontendwizard is suggesting. So, let me know how to proceed. I believe that someone needs to approve the changes I proposed.

**Review by myluco — COMMENTED** [review]
I am not sure how to get this going. I made several comments regarding the "quem possui" does not make sense in brazilian portuguese. 
Not sure what to do next to get the rest of the commit approved

**Pull request closed without merging**